.gitignore

```diff
@@ -242,7 +242,6 @@ _Pvt_Extensions
 
 # Paket dependency manager
 paket-files/
-paket.lock
 paket.bootstrapper.exe

```

Comment:
This line can be removed as well

Reply:
92fdaa3bae09500a265c107c909a346edd8e2fca